### DISCUSSION TITLE
I3WM window management buttons

### DISCUSSION THREAD

**Discussion by NicolasGB** [ghostty-org/ghostty#2807] [Q&A] [answered]
Hi all, 

Is there a way to remove the 3 window buttons? I use i3wm and i'm used to not having them, since usually apps don't have them and it would be nice if they could be removed in ghostty too. Specially because minimizing and expanding do nothing.

Is this possible? haven't seen anything in the config.  

![image](https://github.com/user-attachments/assets/c8bf6afa-0c83-4896-8425-592a1834b519)

Tried setting `gtk-titlebar = false` As someone pointed me too this in the discord but enabling this removes completely the decoration and breaks the theming of the tabs bar.

Here's a video:

https://github.com/user-attachments/assets/99864823-4273-4a39-98f8-1d476dcfce5b

**Comment by chaezwav** [comment]
This might be a ball in the park, but I remember when I was using gnome, gnome tweaks would allow you to enable or disable the close, minimize, and maximize buttons. Maybe try using gnome tweaks? I could be completely wrong though

**Comment by NicolasGB** [comment]
Hey! Thanks for taking the time to answer :smile: Unfortunately it didn't seem to do anything i installed gnome-tweaks, and the buttons already appeared disabled. I've toggled them to see if that could make some difference, but nothing happened.

![image](https://github.com/user-attachments/assets/4957f7c3-2e64-4c40-a1cf-e0394c74d620)

But now that you mention, It seems i have this issue with all  gtk4 apps, gtk2/3 seem to behave as expected but all gtk4 apps have the 3 icons. So maybe i should look into that.

**Comment by NicolasGB** [comment, marked as answer]
Found a solution!

In my gtk4 config found in `.config/gtk-4.0/settings.ini`
I added the following line:
```
gtk-decoration-layout=:
```

And now the result is as expected:
![image](https://github.com/user-attachments/assets/5030fd76-8eb7-45df-8d80-deaf9c8377b7)

Marking this as solved :)
